user-guide/live-traffic/auto-responder/rules-edtor.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Rules Editor
+slug: auto-responder-rules-editor
+tags: auto responder rules, create rules, rule editor, regex rules, actions, match
+publish: true
+position: 20
+---
+
+
+# Rules Editor
+
+The __Rules Editor__ interface enables creating and editing rules. You need to enter a __match rule__ and an __action string__, and Fiddler Everywhere will undertake the action if the request URI matches the __match rule__.
+
+![Auto Responder button](../../../images/livetraffic/autoresponder/autoresponder-main-all.png)
+
+## Create New Rule
+
+1. Click on __Create New Rule__ button in the __Rules Editor__.
+![Create New Rule button](../../../images/livetraffic/autoresponder/autoresponder-editor-create.png)
+2. Enter a __match rule__. See the [match rule section](#match-rules) for more details.
+![New Match Rule](../../../images/livetraffic/autoresponder/autoresponder-create-new-match.png)
+3. Enter an __action string__. See the [action string section](#action-strings) for more details.
+![New Action String](../../../images/livetraffic/autoresponder/autoresponder-create-new-action.png)
+4. Add the newly created rule to the __Queue Viewer__ list of rules.
+
+Tips:
+- Rules are applied in the order that they appear in the __Queue Viewer__. Hit the __Down Arrow__ to demote a rule.
+- You can export the current queue of rules [via the __Export__ button]({%slug auto-responder-queue-viewer%}#export-rules) from the __Queue Viewer__ toolbar.
+- You can import a __*.FARX__ file [via the __Import__ button]({%slug auto-responder-queue-viewer%}#import-rules) to queue rules based on previously captured traffic.
+
+
+## Match Rules
+
+Fiddler Everywhere will accept string literals, regular expressions, and a set of some special match rules as listed below.
+
+### String Literals
+
+- __String Literals__ - Will match string literals (case insensitively).
+- __\*__ - Will match everything.
+- __EXAMPLE__ - Will match URLs of type _http://www.**EXAMPLE**.com/somePath_
+- __path1/__ - Will match URLs of type _http://www.example.com/**path1/**query=example_
+- __query__ - Will match URLs of type _http://www.example.com/path1/q=**query**_
+
+### NOT rules (for String Literals)
+
+The __NOT:__ rule is applied only if the string does not match.
+
+- __NOT:EXAMPLE__ - Will match URLs that are not containing the **EXAMPLE** string.
```

Comment:
For some reason I do not like "are not containing". Consider using "do not contain" instead.